cns/grpc/cnsv1alpha/server_grpc.pb.go

```diff
@@ -0,0 +1,152 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
+// versions:
+// - protoc-gen-go-grpc v1.3.0
+// - protoc             v3.12.4
+// source: cns/grpc/proto/server.proto
+
+package cnsv1alpha
```

Comment:
package name is redundant, doesn't need to restate CNS

Reply:
The package has been renamed to `v1alpha`